internal/weldr/api.go

```diff
@@ -701,11 +704,44 @@ func (api *API) blueprintsInfoHandler(writer http.ResponseWriter, request *http.
 		changes = append(changes, change{changed, blueprint.Name})
 	}
 
-	json.NewEncoder(writer).Encode(reply{
-		Blueprints: blueprints,
-		Changes:    changes,
-		Errors:     []responseError{},
-	})
+	q, err := url.ParseQuery(request.URL.RawQuery)
+	if err != nil {
+		errors := responseError{
+			ID:  "InvalidChars",
+			Msg: fmt.Sprintf("invalid query string: %v", err),
+		}
+		statusResponseError(writer, http.StatusBadRequest, errors)
+		return
+	}
+
+	format := q.Get("format")
+	if format == "json" || format == "" {
+		json.NewEncoder(writer).Encode(reply{
+			Blueprints: blueprints,
+			Changes:    changes,
+			Errors:     []responseError{},
+		})
+	} else if format == "toml" {
+		// lorax concatenates multiple blueprints with `\n\n` here,
+		// which is never useful. Deviate by only returning the first
+		// blueprint.
```

Comment:
Wouldn't it be just as easy to just concatenate them all. Not that I think anyone relies on this, but I could imagine tools asking for many and considering only the first, for instance.

Reply:
Nothing should be relying on that, it was basically a hack to be able to support both output formats with multiple blueprints listed. I agree that an error here is the right thing to do.